core/src/main/scala/org/http4s/jdkhttpclient/JdkHttpClient.scala

```diff
@@ -255,6 +256,7 @@ object JdkHttpClient {
         }
 
         builder.executor(exec)
+        builder.connectTimeout(Duration.ofNanos(org.http4s.client.defaults.ConnectTimeout.toNanos))
```

Comment:
Let's remove this as it is already part of #876?

Reply:
Woops, not sure how that slipped in :)